packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offer-details/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "dedicatedCloud_offer_upgrade_custom_report": "Rapport de sauvegarde personnalisable",
```

name must start with "dedicatedCloud_datacenter_backup"
```suggestion
  "dedicatedCloud_datacenter_backup_offer_upgrade_custom_report": "Rapport de sauvegarde personnalisable",
```

make these changes for all translation names